lib/new_relic/agent/log_event_aggregator.rb

```diff
@@ -153,6 +163,7 @@ def register_for_done_configuring(events)
         events.subscribe(:server_source_configuration_added) do
           @high_security = NewRelic::Agent.config[:high_security]
```

Comment:
@jasonrclark  - I noticed this seems to be the same value as in the `#initialize` method. Do you remember why the instance variable was set twice?

Reply:
My intent there was that I didn't know if the server response could send a different value, so it seemed safest to grab the value after all the configs have settled (which is basically what this event represents).

Reply:
Nice, thanks for clarifying!